python/paddle/fluid/dygraph/dygraph_to_static/loop_transformer.py

```diff
@@ -106,6 +106,8 @@ def get_loop_var_names(self, node):
         return loop_var_names, create_var_names
 
     def visit_Name(self, node):
+        if node.id == 'fluid':
+            return
```

Comment:
What's this line of code for? If users import paddle by ` import fluid.layers as layers` to create tensor by `layers.XXX`, Wether is `if node.id == layers` needed ？

Reply:
This is a temporary way to avoid bug in loop_transformer. The bug is fixed in #22867 . The code above will be delete if #22867 merged.